### PR TITLE
Update NET2.yaml

### PR DESCRIPTION
**Pull request by edubach**
Adding a varnish proxy for NET2 at NRP.

### CODE REVIEW

File: topology/University of Massachusetts - Amherst/NET2/NET2.yaml [hunk 1]
```diff
@@ -153,14 +153,43 @@ Resources:
...
+        Primary:
+          ID: OSG1000443
+          Name: William Leight
+        Secondary:
+          ID: OSG1000284
+          Name: Eduardo Bach
+    Description:
+      This is a NET2 Varnish proxy at NRP. 
+    FQDN: storage-01.nrp.mghpcc.org
+    ID:
```
Comment (brianhlin) [inline topology/University of Massachusetts - Amherst/NET2/NET2.yaml:187]:
```suggestion
```

This will get autogenerated

**Review by brianhlin — APPROVED** [review]
LGTM

File: topology/University of Massachusetts - Amherst/NET2/NET2.yaml [hunk 2]
```diff
@@ -153,14 +153,43 @@ Resources:
...
+    Description:
+      This is a NET2 Varnish proxy at NRP. 
+    FQDN: storage-01.nrp.mghpcc.org
+    ID: 
+    Services:
+      Squid:
+        Description: This is a Varnish proxy flavour.
+        Details:
+          uri_override: storage-01.nrp.mghpcc.org:6082
+          Monitored: true
```
Comment (brianhlin) [inline topology/University of Massachusetts - Amherst/NET2/NET2.yaml:193]: Does the WLCG actually sensibly monitor Varnish proxies? (@ivukotic @DrDaveD )

**Comment by edubach** [comment]
Hi Ilija,

Thanks for the info. I put the bare minimum as I was told to keep it at a minimum, but I can add (or remove) any text you feel fit, maybe some direct mention that it was deployed by you ? Just let me know. I can also add you as the contact.
We will probably see it at work only the site is up again after the downtime.
Once again, thanks a lot for the help.

Eduardo.
________________________________

**Comment by ivukotic** [comment]
Hi Eduardo,

All is fine, there is no need to mention me… only if you want…

Best,
Ilija

**Pull request merged**